guides/common/modules/ref_storage-requirements.adoc

```diff
@@ -30,5 +30,5 @@ endif::[]
 ifdef::foreman-el,katello,satellite[]
 For external database servers: `{postgresql-lib-dir}` with installation size of 100 MB and runtime size of 20 GB.
 
-For detailed information on partitioning and size, see {RHELDocsBaseURL}9/html/performing_a_standard_rhel_9_installation/partitioning-reference_installing-rhel[Partitioning reference] in _Performing a standard RHEL 9 installation_.
+For detailed information on partitioning and size, see {RHELDocsBaseURL}9/html/managing_storage_devices/disk-partitions_managing-storage-devices[Disk partitions] in _Managing storage devices_.
```

Comment:
```suggestion
For detailed information on partitioning and size, see {RHELDocsBaseURL}9/html/managing_storage_devices/disk-partitions_managing-storage-devices[Disk partitions] in _{RHEL-abbr}{nbsp}9 Managing storage devices_.
```
Please, include the product name and version in the guide description for clarity and context.